tutorials/plot_morph_stc.py

```diff
@@ -0,0 +1,248 @@
+
+In MNE-Python, surface source estimates are represented as
+:class:`mne.SourceEstimate` or :class:`mne.VectorSourceEstimate`. Those can
+be used together with :class:`mne.SourceSpaces` or without.
+
+The morph was successful if functional data of Subject A overlaps with
+anatomical surface data of Subject B, in the same way it does for Subject A.
+
+See :ref:`sphx_glr_auto_examples_inverse_plot_morph_surface_stc.py` for an
+example of such a morph.
+
+Setting up SourceMorph for SourceEstimate
+-----------------------------------------
+
+We know that surface source estimates are represented as lists of vertices.
+The respective list of our data can either be obtained from
+:class:`mne.SourceSpaces` (src) or from the data we want to morph itself. If
+src is not provided, the morph will not be precomputed but instead will be
+prepared for morphing when calling.
+
+This works only with (Vector) :class:`SourceEstimate <mne.SourceEstimate>`.
+See :class:`mne.SourceMorph` for additional parameter settings. We keep the
+default parameters for *src* and *spacing*.
+
+Since the default of spacing (resolution of surface mesh) is 5 and subject_to
+was set to 'fsaverage', SourceMorph will use default vertices to morph
+(``[np.arange(10242)] * 2``).
+
+If src was not defined, the morph will actually not be precomputed, because
+we lack the vertices *from* that we want to compute. Instead the morph will
+be set up and when applying it, the actual transformation will be computed on
+the fly.
+
+    >>> morph = SourceMorph(subject_from='sample',  # Default: None
+    >>>                     subject_to='fsaverage',  # Default
+    >>>                     subjects_dir=subjects_dir,  # Default: None
+    >>>                     src=None,  # Default
+    >>>                     spacing=5)  # Default
+
+Apply morph to (Vector) SourceEstimate
+--------------------------------------
+
+The morph will be applied to the source estimate data, by giving it as the
+first argument to the morph we computed above.
+
+    >>> stc_fsaverage = morph(stc)
+
+Example result for morphing **surface** source estimates
+--------------------------------------------------------
+
+:ref:`sphx_glr_auto_examples_inverse_plot_morph_surface_stc.py`
+
+.. image:: ../../_images/sphx_glr_plot_morph_surface_stc_001.png
+
+Reading and writing SourceMorph from and to disk
+================================================
+
+An instance of SourceMorph can be saved, by calling
+:meth:`morph.save <mne.SourceMorph.save>`.
+
+    >>> morph.save('my-file-name')
+
+This methods allows for specification of a filename. The morph will be save
+in ".h5" format. If no file extension is provided, "-morph.h5" will be
+appended to the respective defined filename.
+
+In turn, reading a saved source morph can be achieved by using
+:func:`mne.read_source_morph`:
+
+    >>> from mne import read_source_morph
+    >>> morph = read_source_morph('my-file-name-morph.h5')
+
+Shortcuts
+=========
```

Comment:
I'm a bit scared of this section. According to the Zen of Python:

> There should be one-- and preferably only one --obvious way to do it.

but I suppose you're not Dutch :)

Reply:
I agree with @jasmainak . Is it really just a short cut ? maybe is allows to get a volume without morphing ie staying in the same subject?